missie.md

```diff
@@ -1,41 +1,15 @@
-#Nederlands
+I *Make things*,  
+I work hard, and start things,
+I dance, sing, and drink to beautiful music.
+I either let go of my worries, or I take responsibility.
 
-Ik daag mezelf uit om mooie nuttige dingen te maken,
-en om nieuwe skills te vergaren. 
+I do the above with friends and family 
+but mostly with my wife and children, 
+for whom I provide and protect.
 
-Ik geniet van elk moment,
-alleen en samen met vrienden,
-en familie. 
-Deze kunnen rekenen op mijn hulp en steun,
-en op goede spijzen en dranken.
-
-Mijn omgeving is,
-en de mensen daarin zijn,
-dynamisch, gedreven,
-uitdagend en inspirerend,
-waardoor ik continue kan leren,
-en uitproberen. 
-Zij hebben reden om mij te waarderen,
-te vertrouwen, 
-en tegen mij op te kijken.
-Tegelijkertijd kan ik ook tegen hen opkijken,
-van hen leren, 
-en hen vertrouwen.
-
-#English
-
-I *Make things*, and I acquire new skills.
-
-I enjoy the small things, and the silent moments.
-With friends and family, and mostly with Judith.
-
-My friends and family can expect my help, 
-as well as lots of food and drinks, 
-when they desire it.
-
-The people surrounding me are
+I surround myself with people who are
```

Comment:
Neeee, niet de hele GHG payoff erin! Is zo afgezaagd :-) 

Ik vind: pak daar juist uit wat anderen niet hebben. Iedereen wil dat zijn omgeving leuk is. Niet iedereen vindt dat proactieve belangrijk. Kun je "fun" niet ook scherper maken? Wanneer vind *jij* iemand fun? Ik ken iemand die maakt de hele tijd vunzige grapjes en deelt plaatjes van naakte dames die als televisiemeubel dienen. Die heeft vast vrienden die hem leuk vinden :)